content/akeneo-event-platform/platform-usage-requirements.md

```diff
@@ -13,11 +13,6 @@ In the event that the checkbox is unchecked after a subscription has already bee
 Permissions for the Event Platform are managed at a global level. You cannot assign specific rights. If this global permission is not enabled, the connection will not be able to subscribe to events, and any existing subscription will be suspended if the permission is later revoked.
 
 
-## HTTPS destination
-
-The platform will test the HTTPS endpoint validitybefore being able to create the subscription.
```

Comment:
any specific reason you removed this? (maybe it was added somewhere else and I missed it)

Reply:
For this reason :)
![image](https://github.com/user-attachments/assets/74e3f279-de6d-48df-94c8-7a99078ab52f)

Reply:
oh okay sorry I had missed it!